src/layers/conv.jl

```diff
@@ -1065,4 +1065,117 @@ function (l::MEGNetConv)(g::GNNGraph, x::AbstractMatrix, e::AbstractMatrix)
     return x̄, ē
 end
 
+@doc raw"""
+    GMMConv(in => out, n_kernel, u_dim, σ=identity; [init, bias])
+
+Graph mixture model convolution layer from the paper [Geometric deep learning on graphs and manifolds using mixture model CNNs](https://arxiv.org/abs/1611.08402)
+
+Performs the operation
+```math
+\mathbf{x}_i' = \frac{1}{|N(i)|} \sum_{j\in N(i)}\frac{1}{K}\sum_{k=1}^k\mathbf{w}_k(u_{i,j})\odot \mathbf{\theta}_k \mathbf{x}_j
+```
+
+where
+```math
+\mathbf{w_{k}} = \exp(\frac{-1}{2}(u-\mu_k)^T \Sigma_k^{-1}(u-\mu
+_k))
```

Comment:
```suggestion
\mathbf{w_{k}} = \exp(\frac{-1}{2}(u-\mu_k)^T \Sigma_k^{-1}(u-\mu_k))
```
Also, \mu and \Sigma need to be specified

Reply:
no actually is not a scalar

Reply:
Ok, I edited the suggestion so that now it should be mathematically clearer what's going on

Reply:
Thanks!